### PR TITLE
adjustments made to navbar

### PR DESCRIPTION
**Pull request by dillonstreator**
re-ordered menu
added dropdown menu when user is signed in which contains 'my account', 'my subscriptions', & 'log out'

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -62,5 +60,16 @@
       </div>
     </div>
   </footer>
+  <!-- Dropdown Structure -->
+  <ul id='dropdown1' class='dropdown-content'>
```
Comment (heydanhey) [inline app/views/layouts/application.html.erb]: This is really good. My only suggestion would be that this id is not really descriptive. Could we use `account-dropdown` or something? What do you think?

**Comment by DillonStreator** [comment]
I definitely agree. Will need to be changed in two places. It is referenced
in the actual dropdown action in the navbar

**Comment by DillonStreator** [comment]
I can update this push?
On Saturday, June 24, 2017 at 9:06 AM Dan Hey <notifications@github.com> wrote:

@heydanhey commented on this pull request.

In app/views/layouts/application.html.erb:

> @@ -62,5 +60,16 @@ </div> </div> </footer> +  + <ul id='dropdown1' class='dropdown-content'> 

This is really good. My only suggestion would be that this id is not really descriptive. Could we use account-dropdown or something? What do you think?

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (dillonstreator) [reply app/views/layouts/application.html.erb]: I agree =}

**Comment by dillonstreator** [comment]
More descriptive ID name =)

**Review by heydanhey — APPROVED** [review]
+1

**Pull request merged**